_data/docs.yml

```
@@ -46,6 +46,17 @@ apis:
       legacy: 1
       stable: 1
       nightly: 1
+  nx-cugraph:
+    name: nx-cugraph
+    path: cugraph # TODO: determine where to point nx-cugraph api docs
+    desc: 'nx-cugraph is a NetworkX backend that provides GPU acceleration to many popular NetworkX algorithms. With nx-cugraph, users can have GPU-based, large-scale performance without needing to change their pre-existing NetworkX code.'
+    ghlink: https://github.com/rapidsai/nx-cugraph
+    cllink: https://github.com/rapidsai/nx-cugraph/blob/main/CHANGELOG.md
+    versions:
+      # enable or disable links; 0 = disabled, 1 = enabled
+      legacy: 0
+      stable: 0
```

At this point we do have legacy and stable docs for nx-cugraph because it has existed since 24.06.

Legacy: https://docs.rapids.ai/api/cugraph/legacy/nx_cugraph/
Stable: https://docs.rapids.ai/api/cugraph/stable/nx_cugraph/
```suggestion
      legacy: 1
      stable: 1
```